lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/TypeProcessor.java

```diff
@@ -53,6 +55,17 @@ protected boolean handleProcess(
     for (TypeElement annotation : annotations) {
       Set<? extends Element> annotatedElements = roundEnv.getElementsAnnotatedWith(annotation);
       for (Element elt : annotatedElements) {
+        if (jfo == null) {
+          // Create generator for Java source file once(!) so that it can be used at the last
+          // round of processing. Otherwise, javac complains that the generated file won't be
+          // used in further processing. That's fine, we know it won't.
+          try {
+            jfo = processingEnv.getFiler().createSourceFile(ConstantsGenFullClassname);
```

Comment:
Why are we setting `jfo` in a loop, and not in the class constructor for example?

Reply:
It's not particularly happy about it
```
[error] error: Bad service configuration file, or exception thrown while constructing Processor object: javax.annotation.processing.Processor: Provider org.enso.interpreter.dsl.TypeProcessor could not be instantiated
```

Reply:
Just to maybe expand on it, it's particularly happy about using the processing env in the constructor, it seems

Reply:
I see, the usual initialization issues. But can we maybe set it once in this function, just outside of loop?

Reply:
:+1: